README.adoc

```diff
@@ -28,19 +28,20 @@ You'll explore how to manage the impact of failures using MicroProfile Fault Tol
 You will learn how to build fault-tolerant microservices in an inventory management application so
 that you can reduce the impact from failure and ensure continued operation of services.
 
-The application that you will be working with is an `inventory` service, which stores the information
-about various JVMs that run on different systems. When you make a request to the `inventory` service
-to retrieve the JVM system properties of a particular host, the `inventory` service communicates with
-the `system` service on that host to get these system properties.
-The `inventory` service then stores and returns the system properties.
+The application that you will be working with is an `inventory` service, which collects,
+stores and returns the system properties.
+It uses the `system` service to retrieve the system properties for a particular host.
+You will add fault tolerance to the `inventory` service so that it reacts accordingly with the `system`
+service being unavailable.
```

Comment:
How about..."so that it reacts accordingly **when** the `system` service **is** unavailable."

Reply:
Fixed